### PR TITLE
Added createjs namespace so the examples work as is

### PR DESCRIPTION
**Pull request by cotejp**

### CODE REVIEW

File: src/tweenjs/Ease.js [hunk 1]
```diff
@@ -42,11 +42,11 @@ this.createjs = this.createjs||{};
 	 *
 	 * Most methods on Ease can be passed directly as easing functions:
 	 *
-	 *      Tween.get(target).to({x:100}, 500, Ease.linear);
+	 *      Tween.get(target).to({x:100}, 500, createjs.Ease.linear);
```
Comment (lannymcnie) [inline src/tweenjs/Ease.js]: Any chance you can also change the `Tween.get` usages to `createjs.Tween.get` as well? Appreciate the catch!

Comment (cotejp) [reply src/tweenjs/Ease.js]: Oh my god! I must be going blind... I fixed it.
Cheers!

**Comment by cotejp** [comment]
Do you happen to know at which frequency the docs on createjs.com are updated? I basically fixed this little issue so my students would stop nagging me with the same question over and over... :wink:

**Comment by lannymcnie** [comment]
We don't typically update the NEXT docs until we release a new stable version. I would be happy to push up the latest today though, not a ton has changed since the 1.0.0 release.

Cheers

**Comment by cotejp** [comment]
If that's easy for you, it would be great. If it takes more than 5 minutes, just forget it. It's no big deal. Thanks again!

**Comment by lannymcnie** [comment]
I updated the site to host the latest TweenJS docs :)

Cheers.

**Comment by cotejp** [comment]
Awesome! Thanks.

**Pull request merged**